backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/DebugUtils.kt

```diff
@@ -23,18 +24,39 @@ import org.jetbrains.kotlin.types.TypeUtils
 internal object DWARF {
     val producer                       = "konanc ${KonanVersion.CURRENT} / kotlin-compiler: ${KotlinVersion.CURRENT}"
     /* TODO: from LLVM sources is unclear what runtimeVersion corresponds to term in terms of dwarf specification. */
-    val runtimeVersion                 = 2
     val dwarfVersionMetaDataNodeName   = "Dwarf Version".mdString()
     val dwarfDebugInfoMetaDataNodeName = "Debug Info Version".mdString()
-    val dwarfVersion = 2 /* TODO: configurable? like gcc/clang -gdwarf-2 and so on. */
-    val debugInfoVersion = 3 /* TODO: configurable? */
     /**
      * This is  the value taken from [DIFlags.FlagFwdDecl], to mark type declaration as
      * forward one.
      */
-    val flagsForwardDeclaration = 4
+    const val flagsForwardDeclaration = 4
+
+    fun debugInfoVersion(config: KonanConfig):Int = 3
```

Comment:
Why not val?

Reply:
I've thought about making this parameter configurable but for now we can leave it as val.